### DISCUSSION TITLE
[BUG] Possible incorrect CURL command generation

### DISCUSSION THREAD

**Discussion by natepisarski** [spatie/flareapp.io-roadmap#69] [Ideas]
Hi, I'm not sure if this is the proper avenue to report a bug - please feel free to delete/move if it's not.

I noticed that if you have a situation like this:

**POST /api/blah** `application/json`
```json
{
  "someObject": {
     "something": 1
  }
}
```

then Flare will generate a CURL request that looks like:

```bash
curl "https://flowbirdhub.com/api/v1/axio/manifest" \
   -X POST \
  -F 'someObject=[object Object]'
```

Obviously `[object Object]` is what appears when Javascript casts an object to a string.

I'm not familiar enough with CURL to know what the correct command is, but from very quick research it looks like this case should generate a `-d` flag instead.

**Comment by AlexVanderbist** [comment]
I'll recreate this issue on our internal bug tracker. Thanks for reporting this 👍
